src/Datagrid/Pager.php

```diff
@@ -39,9 +39,10 @@ public function computeNbResult()
         }
 
         $countQuery->select(sprintf(
-            'count(%s %s) as cnt',
+            'count(%s %s.%s) as cnt',
             $countQuery instanceof ProxyQuery && !$countQuery->isDistinct() ? null : 'DISTINCT',
```

Comment:
Distinct is always set on 3.5.1: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.5.1/src/Datagrid/Pager.php#L40

From where come from this change?

Reply:
Distinct was added by this commit https://github.com/sonata-project/SonataAdminBundle/commit/c7f454ed86b0df5f636af90cff5cbac6cad45953#diff-21e8075cdf497941b782a6a3311751cb

Reply:
Yes, but why the ternary condition here? Not related?

I'm currently trying to install your fork on my "proof project".

Reply:
To preserve the feature with performance #827